cli/provider/cmd.go

```diff
@@ -182,11 +182,13 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 	case "upload": //for uploading mocks
 		cmd.Flags().StringP("path", "p", ".", "Path to local keploy directory where generated mocks are stored")
 		cmd.Flags().StringSliceP("test-sets", "t", utils.Keys(c.cfg.Test.SelectedTests), "Testsets to consider e.g. -t \"test-set-1, test-set-2\"")
-	case "generate", "download":
 
+	case "generate", "download":
 		if cmd.Name() == "download" && cmd.Parent() != nil && cmd.Parent().Name() == "mock" { // for downloading mocks
 			cmd.Flags().StringP("path", "p", ".", "Path to local keploy directory where generated mocks are stored")
 			cmd.Flags().StringSliceP("test-sets", "t", utils.Keys(c.cfg.Test.SelectedTests), "Testsets to consider e.g. -t \"test-set-1, test-set-2\"")
+			cmd.Flags().String("registry-id", "", "Registry ID for direct mock download (e.g., xxxx-xxxx-xxxx)")
+			cmd.Flags().String("app", "", "App name (defaults to current directory name if not provided)")
```

Comment:
The appName flag is already there, and it is not coupled with any command. Please use that. You don't have to make an app flag for the mock download command.